### DISCUSSION TITLE
More Functions, and "Insert Row" between functions/switches.

### DISCUSSION THREAD

**Discussion by EhAye** [EdgeTX/edgetx#2586] [Ideas]
- Copy timers using a Special Function. I can only seem to set it to a specific value. I want to copy Timer1 to Timer3. 

- Set the 6POS position via Special Function.

- Insert a Logical Switch between two others, (shifting other switches down).  Bonus if it automatically modifies references to the shifted switches/functions, to point to their new location.

and it would be great if we could label the functions and switches. Maybe not for reference in the menus, but just as comments so i remember which of these does what.

**Comment by raphaelcoeffic** [comment]
I would rather propose you use a mixer LUA script for that purpose. The 6POS you want to set would be a virtual channel output created by the script.

**Reply by raphaelcoeffic** [reply]
Regarding the labels, it is on our TODO list, just not yet for a specific version.

**Reply by pfeerick** [reply]
Lables for LS and SF is hidden in this PR ;) #2085 And in the case of the T16/TX16S/T18, setting the 6POS position isn't going to be possible, since the TX has no control over the buttons, and thus the 6POS state.
